android_splash_screen/README.md

```diff
@@ -0,0 +1,34 @@
+# Splash Screen Sample
+
+A Flutter app that exemplifies how to implement an animated splash screen for Android devices running at least Android 12, the version that supplies the new [SplashScreen API](https://developer.android.com/about/versions/12/features/splash-screen).
+
+**NOTE:** There is a pub package available to implement static splash screens in your Flutter app: [flutter_native_splash](https://pub.dev/packages/flutter_native_splash).
+
+
+## Goals
+- Demonstrate the compatibility of animated splash screens and Flutter apps running on Android
+- Demonstrate the smoothness achievable by a splash screen as a transition to the Flutter UI
+
+## The Important Bits
+
+### Remove deprecated code
+
+When creating a Flutter app, the Android code generated may include the deprecated implementation of a splash screen. This includes a definition of `io.flutter.embedding.android.SplashScreenDrawable` in the ` /android/app/src/main/AndroidManifest.xml` file and an implementation of `provideSplashScreen()` in the `/android/app/src/main/kotlin/MainActivity.kt` file. Make sure to remove this code.
+
+### Modify Android build files
+In order to support the Android 12 SplashScreen API, you will need to:
```

Comment:
nit: Remove "will" in "you will need to." The tech writers always tell us to stick with the present tense in these situations.